### DISCUSSION TITLE
Everything stops after reaching "info   Startup time:"

### DISCUSSION THREAD

**Discussion by OldManGir42** [vladmandic/sdnext#1530] [Q&A] [answered]
After quite some time working on getting things going I finally got the .bat to run, but when it gets to "INFO      Startup time:" everything just stops, and quite frankly I'm not even sure what's supposed to come next, but it's been sitting for about half an hour now and I've never reached a UI...

Can anyone help?

**Comment by vladmandic** [comment, marked as answer]
thats it - there is nothing more, its ready for you to fire up browser.
if you look at messages just prior, you'll see local url printed.
or you can use `--autolaunch` flag to automatically start browser.

**Reply by OldManGir42** [reply]
Awesome, thanks! Now to figure out how to get it to use my 6900xt...

**Reply by brknsoul** [reply]
In windows? add `--use-directml`

**Reply by OldManGir42** [reply]
I've tried that. As it is, I'm having trouble getting torch_directml to be selected. On loading it keeps loading cpu-only torch and torch 2.0.1+cpu instead off the directml variant, even when I use that argument.

edit to clarify: I have run uninstall for any and every version of torch, and I can confirm that I have (finally, after much frustration) torch_directml installed

**Reply by brknsoul** [reply]
If you've ran webui.bat for the first time without `--use-directml`, then SD.Next has only installed the CPU torch.

You'll need to run `webui.bat --use-directml --reinstall`

If you've f-d around with manually installing stuff, your best best is to nuke the folder and start again.

Your console should read something like
```
Using DirectML Backend
Torch 2.0.0+cpu
Torch backend: DirectML (torch-directml 0.2.0.dev230426)
```

**Reply by OldManGir42** [reply]
By Jove, that's done it! Thanks a ton, friend!

**Comment by brknsoul** [comment]
Hey, Vlad.. perhaps quick suggestion; if `--autolaunch` not on command line, perhaps a little line after "Startup time"
"SD.Next is ready, use your internet browser with URL \<ip\> to begin!" or something ;-)
